doc/cygnus-twitter/to_be_done.md

```diff
@@ -0,0 +1 @@
+#To be done
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/949/commits/540563c64137c0bf10bcebaa9cf8f4dfe26093e1